_rules/id-value-unique-3ea0c8.md

```diff
@@ -70,7 +70,7 @@ All `id`s are unique within the document context.
 
 #### Passed Example 3
 
-Two of the `id` are the same (`my-elt`), but they are in different trees (the first one in the [document tree](https://dom.spec.whatwg.org/#document-trees) and the second in the [shadow tree](https://dom.spec.whatwg.org/#shadow-trees)).
+Two of the `id`s are the same (`my-elt`), but they are in different trees (the first one in the [document tree](https://dom.spec.whatwg.org/#document-trees) and the second in the [shadow tree](https://dom.spec.whatwg.org/#shadow-trees)).
```

Comment:
I don't think the plural is necessary, as 'id' can be considered implicitly plural.

Reply:
OK. Between @daniel-montalvo (https://github.com/act-rules/act-rules.github.io/pull/1146#pullrequestreview-352979261) and you, I'll trust the native speaker…